cartridges/int_stickyio_sfra/cartridge/templates/default/checkout/shipping/shipping.isml

```diff
@@ -1,33 +1,41 @@
-<div class="shipping-section">
+<isif condition="${pdict.sfccVersion60}">
+	<div class="card ghost shipping">
+	    <div class="card-header">
+	        <h2 class="card-header-custom">${Resource.msg('heading.checkout.shipping', 'checkout', null)}</h2>
+	    </div>
+	</div>
+</isif>
+<div class="<isif condition="${pdict.sfccVersion60}">card </isif>shipping-section">
     <div class="single-shipping">
         <isset name="shippingModel" value="${pdict.order.shipping[0]}" scope="page" />
         <isset name="shipmentLoopState" value="${null}" scope="page" />
         <isset name="lineItem" value="${null}" scope="page" />
-        <div class="card" data-shipment-uuid="${shippingModel.UUID}">
+        <div  <isif condition="${!pdict.sfccVersion60}"> class="card" </isif> data-shipment-uuid="${shippingModel.UUID}">
             <div class="card-header">
                 <h2 class="card-header-custom">${Resource.msg('heading.checkout.shipping', 'checkout', null)}</h2>
             </div>
 
             <div class="shipping-error"></div>
-            <isif condition="${!!pdict.preventMultiShip === false}">
-                <isinclude template="checkout/shipping/multiShipCheckbox" />
-            </isif>
+			 <isif condition="${!!pdict.preventMultiShip === false}">
+            	<isinclude template="checkout/shipping/multiShipCheckbox" />
+			</isif>
```

Comment:
Please correct spacing issues in lines 19-21?

Reply:
fixed